cea/optimization/slave/cooling_resource_activation.py

```diff
@@ -20,30 +20,30 @@
 __status__ = "Production"
 
 
-def calc_vcc_operation(Qc_from_VCC_W, T_DCN_re_K, T_DCN_sup_K, prices, lca, limits):
+def calc_vcc_operation(Qc_from_VCC_W, T_DCN_re_K, T_DCN_sup_K, prices, lca, limits, hour):
```

Comment:
here as well: we're not interested in the hour at all. or the lca object. all we really care about is  the `electricity_price` (for the hour - but what the `calc_vcc_operation` calculation does not depend on the hour at all!

Reply:
It is true that the hour is not actually necessary, but in `lca`, we need `EL_TO_CO2`, `EL_TO_OIL_EQ` and `ELEC_PRICE`. Only electricity price is hour dependent

One way to do this is pass all these variables in the function call. Other way to do this is pass the lca fully and then also pass the hour. 

Is one way better than the other? I am not so sure.